### ISSUE TITLE
Header Tooltips incorrect

### ISSUE THREAD

**Issue by WorldsEndless** [BYU-ODH/wp-humanities-byu#150]
![image](https://user-images.githubusercontent.com/5042778/109693335-c3edc980-7b46-11eb-9db6-f4f36c3a9195.png)
The tooltip for BYU shows College of Humanities.

![image](https://user-images.githubusercontent.com/5042778/109693413-dc5de400-7b46-11eb-913a-29346c73c84b.png)

![image](https://user-images.githubusercontent.com/5042778/109693437-e384f200-7b46-11eb-94eb-e9ad171c01af.png)

The tooltip for CoH shows the Think Clearly... slogan (typo)

These need to be fixed or removed. They should be accessibility-friendly tooltips, meaning giving text that screen readers can read for those who can't actually see the buttons.
